### ISSUE TITLE
Question regarding meta's and passing fields to traces

### ISSUE THREAD

**Issue by easterbrennan** [grafana/faro-web-sdk#116]
Hi,  I've been playing around with the demo locally.  I wanted to get some more fields passed through to trace so I tried changing the agent.yml for the trace config to add attributes like this:

      attributes:
        actions:
          - action: upsert
            key: testNewValue
            from_context: meta.user.email

But it doesn't appear to work.  What am I doing wrong or is this even possible / am I approaching this the wrong way?  I'm totally new to this so feel free to set me straight.

Basically my logic was rather than only being able to search on the session_id it would be great to have something like the user_email passed through to tempo and then I could search all traces which include that.

Any thoughts appreciated.

**Comment by codecapitano** [comment]
Hi @easterbrennan thank you for reaching out.

It is not possible to configure the agent to inject meta attributes to spans. 

This needs to be configured via the Faro web-sdk.
Unfortunately we currently only provide functionality to add the `session_id` meta attribute to spans. 
Midterm we will provide an extended processor which can add any meta attribute to spans.

Quick solution for now would be to implement and use your own span processor to add any attributes you like to the spans. 
You can use the [FaroSessionSpanProcessor](https://github.com/grafana/faro-web-sdk/blob/109-otel-transport/packages/web-tracing/src/sessionSpanProcessor.ts#L17) as an example implementation.

Here are the [docs on how to integrate the span processor into the web-sdk](https://grafana.com/docs/grafana-cloud/faro-web-sdk/opentelemetry-js/#integrate-existing-opentelemetry-js-instrumentation-with-faro-web-sdk)

Thanks and hope this may help.

Cheers,
Marco

**Comment by easterbrennan** [comment]
@codecapitano ok cool... thanks for the response, we will take a look at your suggestions.  Have a nice day.
